_observing-your-data/forecast/security.md

```diff
@@ -0,0 +1,462 @@
+
+
+- `forecast_full_access` – For users responsible for managing the full lifecycle of forecasters, including creating, editing, starting, stopping, and deleting them. This role does **not** grant access to the source index. To create a forecaster, users must also have index-level permissions that include the `search` action on any index or alias the forecaster reads from.
+
+The following example shows how these roles are defined:
+
+```yaml
+forecast_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/plugin/forecast/forecaster/info'
+    - 'cluster:admin/plugin/forecast/forecaster/stats'
+    - 'cluster:admin/plugin/forecast/forecaster/suggest'
+    - 'cluster:admin/plugin/forecast/forecaster/validate'
+    - 'cluster:admin/plugin/forecast/forecasters/get'
+    - 'cluster:admin/plugin/forecast/forecasters/info'
+    - 'cluster:admin/plugin/forecast/forecasters/search'
+    - 'cluster:admin/plugin/forecast/result/topForecasts'
+    - 'cluster:admin/plugin/forecast/tasks/search'
+  index_permissions:
+    - index_patterns:
+        - 'opensearch-forecast-result*'
+      allowed_actions:
+        - 'indices:admin/mappings/fields/get*'
+        - 'indices:admin/resolve/index'
+        - 'indices:data/read*'
+
+forecast_full_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/plugin/forecast/*'
+    - 'cluster:admin/settings/update'
+    - 'cluster_monitor'
+  index_permissions:
+    - index_patterns:
+        - '*'
+      allowed_actions:
+        - 'indices:admin/aliases/get'
+        - 'indices:admin/mapping/get'
+        - 'indices:admin/mapping/put'
+        - 'indices:admin/mappings/fields/get*'
+        - 'indices:admin/mappings/get'
+        - 'indices:admin/resolve/index'
+        - 'indices:data/read*'
+        - 'indices:data/read/field_caps*'
+        - 'indices:data/read/search'
+        - 'indices:data/write*'
+        - 'indices_monitor'
+```
+{% include copy.html %}
+
+These roles do not include default `index_permissions` for specific source or result indexes. This is intentional, allowing you to add your own patterns based on your data access requirements.
+
+### Data-source read role
+
+Each forecaster uses the creating user’s credentials to query the source index. To enable this, you must grant that user read permissions on your own data index.
+
+The following example creates a minimal role that allows read access to the `network-metrics` index:
+
+```json
+PUT _plugins/_security/api/roles/data_source_read
+{
+  "index_permissions": [{
+    "index_patterns": ["network-metrics"],
+    "allowed_actions": ["read"]
+  }]
+}
+```
+
+You can modify the `index_patterns` to match your actual data source.
+
+### Result‑read role
+
+The result-read role allows users to view forecast results and configure Alerting monitors that query those results.
```

Comment:
Is this intentionally hyphenated? Should the name of the role be formatted differently (in code font)?